(op-reg) General cleanup of the operation registry documentation. (#334)
apollographql/apollo: docs/source/platform/operation-registry.md

  * To get started with Apollo Server, visit [its documentation](/docs/apollo-server/).
* A client application which utilizes `gql` tagged template literals for its operations or, alternatively, stores operations in `.graphql` files.
* An Apollo Engine API key.
  * To obtain an API key, visit [Apollo Engine](https://engine.apollographql.com) and create a service.

### Installation steps

> Make sure you've met the requirements for _Prerequisites_ above.

These installation steps require access to both the client and server codebases to perform the following tasks:

* The `apollo` CLI is used to search the client codebase for GraphQL operations and upload them to Apollo Engine.
* Apollo Server is then configured with a plugin which fetches the manifest from Apollo Server and enforces safe-listing using that manifest.

The following steps will walk through the steps necessary for both the client and server codebases.

**1. Install the `apollo` command line tool as a development dependency of your client application:**

```
npm install apollo --save-dev
```

> Yarn users should run `yarn add apollo --dev`.

> **2. Register the server's schema with Apollo Engine:**

Reviewer: ```suggestion
**2. Push your schema to the Apollo registry:**
```

I try to shy away from describing the schema registry as a feature of Engine because many people associate Engine with our paid plans even though the schema registry is free. Additionally, I think associating it with Engine instead of the platform goes against what we're trying to achieve with the overall platform story. What do you think?

Author: @peggyrayzis Could we use your suggestion, but still call it the `schema registry` (we have two registries, and I think it's worth clarifying which one we're talking about, specifically in this article which touches on both)?  E.g.:

```suggestion
**2. Push your schema to the Apollo schema registry:**
```

Reviewer: I like that suggestion!
